packages/bpk-component-boilerplate/src/BpkBoilerplate.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { cssModules } from 'bpk-react-utils';
+import BpkText from 'bpk-component-text';
+
+import STYLES from './BpkBoilerplate.scss';
+
+const getClassName = cssModules(STYLES);
+
+type Props = {
+  title: string,
```

Comment:
Do we really need to have `title` and `alternate` here? It feels like we will be removing this most of the times.

Reply:
Bear in mind that this package serves two purposes:

1) To be useful for us.
2) To show contributors our conventions and how we structure code.

In this case I figured this would be a good way to show contributors how we use Flow types and use props. Maybe that's too much hand-holding though, what do you think?

Reply:
I agree with that but I think className is enough to show it. In any case I don't have a strong opinion about this so if you think we should keep it that's fine.